paymentFlow/.juspay/paymentFlowRules.yaml

```diff
@@ -0,0 +1,60 @@
+rules:
+  - type_name: "MerchantAccount"
```

Comment:
this is a public repo, can we remove all these and put a simple non-juspay rule here.